src/stylesheets/components/_components.hero.scss

```diff
@@ -0,0 +1,22 @@
+.c-hero {
+  background-image: url("../images/hero-mobile.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 690px;
+  @include mq($from:tablet) {
+    background-image: url("../images/hero-desktop.png");
+    height: 600px;
+  }
+}
+
+.c-hero__text {
+  padding: 128px 21px 0 12px;
+  @include mq($from:tablet) {
+    padding: 165px 0 0 0;
+  }
+  h1,
+  p {
```

Comment:
Apply this to the element `.c-hero__text` otherwise it's a never-ending addition of elements 🌮

Reply:
The rule of targeting elements within a component class should only be for elements that do not have other options.

For example
```
li
a
img
```